modin/pandas/series.py

```diff
@@ -1022,7 +1022,11 @@ def duplicated(self, keep="first") -> Series:  # noqa: PR01, RT01, D200
         """
         Indicate duplicate Series values.
         """
-        return self.to_frame().duplicated(keep=keep)
+        name = self.name
+        result = self.to_frame().duplicated(keep=keep)
+        # DataFrame.duplicated drops the name, so we need to manually restore it
+        result.name = name
```

Comment:
Since this is a fairly heavy operation that triggers remote computations
```suggestion
        if name is not None:
            result.name = name
```

Reply:
Good idea. Alternatively, we could set the name from within the query compiler to avoid this extra remote computation altogether: this should be easy to do in `PandasQueryCompiler` but might be a bit more annoying in `BaseQueryCompiler`.